### PR TITLE
Add Open Science Labs

### PR DESCRIPTION
**Pull request by xmnlab**
This PR aims to add Open Science Labs as a participating organization for GSoC 2023.
thank you

### CODE REVIEW

**Comment by xmnlab** [comment]
mm for some reason it has more changes that was intended for that. I will take a look into that.

File: README.md [hunk 1]
```diff
@@ -25,14 +25,10 @@ application for Google Summer of Code in the current and previous years.
...
 
-- [Students](#students)
-- [Sub Organizations](#sub-organizations)
-- [Organizations Confirmed Under NumFOCUS Umbrella](#organizations-confirmed-under-numfocus-umbrella)
-- [NumFOCUS Projects](#numfocus-projects)
-    - [Fiscally Sponsored Projects GSoC Status](#fiscally-sponsored-projects-gsoc-status)
-    - [Affiliated Projects GSoC Status](#affiliated-projects-gsoc-status)
-    - [Other Projects GSoC Status](#other-projects-gsoc-status)
-- [About GSoC](#about-gsoc)
+- [Google Summer of Code](#google-summer-of-code)
```
Comment (xmnlab) [inline README.md:28]: it seems it was automatically changed by vscode XD sorry about that

**Comment by xmnlab** [comment]
my IDE also removde the extra whitespace at the end of few lines. let me know if that is a problem I can fix that

File: 2023/ideas-list.md [hunk 2]
```diff
@@ -27,5 +27,6 @@ page of each organization under the NumFOCUS umbrella at this page.
 - [SciML](https://sciml.ai/dev/#google_summer_of_code)
 - [Taskflow](https://github.com/taskflow/GSoC2023)
 - [TNL](https://gitlab.com/tnl-project/tnl/-/wikis/GSoC-2023)
+- [Open Science Labs](https://hackmd.io/@GvoPVECJQAmNl6JmM2UPMQ/BJEITVBAj)
```
Comment (henrykironde) [inline 2023/ideas-list.md:31]: @xmnlab please organize it according to the sorted order used

Comment (xmnlab) [reply 2023/ideas-list.md:31]: oh that is right .. thanks for point that .. working on that right now.

Comment (xmnlab) [reply 2023/ideas-list.md]: @henrykironde I think that it is fixed now. thank you for your review.

**Comment by xmnlab** [comment]
thanks @MridulS !

**Pull request merged**